app/views/directives/notifications/notification-counter.html

```diff
@@ -0,0 +1,13 @@
+<a
+  ng-if="!$ctrl.hide"
+  href=""
+  ng-click="$ctrl.onClick()"
+  class="nav-item-iconic relative">
+  <div
+    ng-if="$ctrl.showNewNotificationIndicator"
+    class="notification-counter notification-counter-offset"
+    row main-axis="center" cross-axis="center">
+    <span><!--{{countDisplay}}--></span>
+  </div>
+  <span class="fa fa-bell" aria-hidden="true"></span>
```

Comment:
@jeff-phillips-18 Agree, but I see some problems with the example as well. Can we fix in Patternfly?

* `a` needs an `href` or you can't access it by the keyboard or screen reader
* `fa-bell` should have `aria-hidden="true"`
* Needs some `sr-only` text for screen readers